conftest.py

```diff
@@ -377,15 +205,15 @@ def t3k_device_mesh(request, silicon_arch_name, silicon_arch_wormhole_b0, device
     except (ValueError, AttributeError):
         num_devices_requested = len(device_ids)
 
+    request.node.pci_ids = [ttl.device.GetPCIeDeviceID(i) for i in device_ids[:num_devices_requested]]
```

Comment:
Damn kinda shitty - you have to `pci_ids` for every fixture that creates any set of devices for any test?

Didn't think this would be an issue, but looks like you'll have to do it.

Is it possible to move this to a higher up fixture like `device_params`, or is that too much?

Reply:
Might be too much, I'd have to add to `device_params` for every pytest no? Or am I misunderstanding how that fixture works.

Reply:
My understanding is that all silicon tests use `device_params` or some fixture downstream, so every test will include it.

Are there cases where silicon tests don't use the regular `device` fixture or the other mesh fixtures ? (Even if the answer to this question is yes, that still means you aren't capturing certain tests with your PR, here)